www/package.json

```diff
@@ -3,6 +3,9 @@
   "description": "JS files required for the Wikipedia app",
   "repository": "https://gerrit.wikimedia.org/r/apps/ios/wikipedia",
   "version": "0.1.0",
+  "dependencies": {
+       "applib": "0.1.2"
```

Comment:
Consider merging [this guy](https://github.com/wikimedia/applib/pull/1) and then you can use the new API